source/components/organisms/Step/Step.js

```diff
@@ -41,6 +41,7 @@ const SignStepWrapper = styled.View`
 
 function Step({
   theme,
+  navigation,
```

Comment:
Okay, so minor thing, but I think we can and should avoid this prop drilling of the navigation prop, and instead use the useNavigation hook as close to the use of it as we can, i.e. in StepConfirmation. To get the navigation object, you can do
import { useNavigation } from '@react-navigation/native';
and
const navigation = useNavigation();

Reply:
Yea that was much better. Now the dirt is isolated to the "step confirmation" view and can easily be removed when a better method is implemented.